src/app/App.js

```diff
@@ -18,6 +19,13 @@ const Grid = styled.div`
 
 export default function App() {
   const [recipesList, setRecipesList] = useState(getLocal('recipesList') || []);
+  const [favorites, setFavorites] = useState(
+    getLocal('favoritesList') ||
+      recipesList.map(recipe => ({
```

Comment:
This line seems to be without a purpose as the initial local storage has an empty array and not the expected default values in an array.
<img width="466" alt="Bildschirmfoto 2019-06-04 um 14 32 06" src="https://user-images.githubusercontent.com/48948963/58879391-df0f7900-86d5-11e9-8be1-d48163c575d7.png">

Reply:
has been deleted